torch/_torch_docs.py

```diff
@@ -4181,6 +4181,20 @@
 
 """)
 
+add_docstr(torch.randn_like,
+           r"""
+randn_like(other) -> Tensor
```

Comment:
This also takes in `device` and `dtype`, can you add those as well? If given, they overwrite the device and dtype of `other` iirc.

Reply:
Generally all the tensor factory methods need this treatment...

Reply:
Sure. It looks like it doesn't take a `device` keyword though.

Reply:
It should, you may have to update your build.

```
    "randn_like(Tensor input, *, ScalarType dtype=None, Layout layout=None, Device device=None, bool requires_grad=False)",
```

Reply:
Yes; I would just look at the generated code instead of trying to figure out the generation rules.